azuredevops/internal/acceptancetests/resource_workitemquery_permissions_test.go

```diff
@@ -0,0 +1,194 @@
+%s
+
+data "azuredevops_group" "project-administrators" {
+	project_id = azuredevops_project.project.id
+	name       = "Project administrators"
+}
+
+resource "azuredevops_workitemquery_permissions" "wiq-permissions" {
+	project_id  = azuredevops_project.project.id
+	principal   = data.azuredevops_group.project-administrators.id
+	%s
+	permissions = {
+		%s
+	}
+}
+`, projectResource, szPath, szPermissions)
+}
+
+func TestAccWorkItemQueryPermissions_SetProjectPermissions(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	permissions := map[string]string{
+		"Contribute":        "Allow",
+		"Delete":            "Deny",
+		"ManagePermissions": "NotSet",
+	}
+	config := hclWorkItemQueryPermissions(projectName, "", permissions)
+
+	tfNode := "azuredevops_workitemquery_permissions.wiq-permissions"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: testutils.CheckProjectDestroyed,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testutils.CheckProjectExists(projectName),
+					resource.TestCheckResourceAttrSet(tfNode, "project_id"),
+					resource.TestCheckResourceAttrSet(tfNode, "principal"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.%", "3"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.Contribute", "allow"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.Delete", "deny"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.ManagePermissions", "notset"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccWorkItemQueryPermissions_UpdateProjectPermissions(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	config1 := hclWorkItemQueryPermissions(projectName, "", map[string]string{
+		"Contribute":        "Allow",
+		"Delete":            "Deny",
+		"ManagePermissions": "NotSet",
+	})
+	config2 := hclWorkItemQueryPermissions(projectName, "", map[string]string{
+		"Contribute":        "Deny",
+		"Delete":            "Allow",
+		"ManagePermissions": "Deny",
+	})
+
+	tfNode := "azuredevops_workitemquery_permissions.wiq-permissions"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: testutils.CheckProjectDestroyed,
+		Steps: []resource.TestStep{
+			{
+				Config: config1,
+				Check: resource.ComposeTestCheckFunc(
+					testutils.CheckProjectExists(projectName),
+					resource.TestCheckResourceAttrSet(tfNode, "project_id"),
+					resource.TestCheckResourceAttrSet(tfNode, "principal"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.%", "3"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.Contribute", "allow"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.Delete", "deny"),
+					resource.TestCheckResourceAttr(tfNode, "permissions.ManagePermissions", "notset"),
+				),
+				Destroy: false,
```

Comment:
`Destroy=false` can  be removed, `Destroy` is set to false by default.

Reply:
Removed per 71fd32738a18d0413fa5f54225e17e483b6880d4